clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_cli.py

```diff
@@ -16,13 +16,18 @@
 @cli_param.option.subjects_sessions_tsv
 @cli_param.option.working_directory
 @cli_param.option.n_procs
+@cli_param.option_group.advanced_pipeline_options
+@cli_param.option.use_cuda
+@cli_param.option.initrand
 def cli(
     bids_directory: str,
     caps_directory: str,
     low_bval: int = 5,
     subjects_sessions_tsv: Optional[str] = None,
     working_directory: Optional[str] = None,
     n_procs: Optional[int] = None,
+    use_cuda: Optional[bool] = False,
```

Comment:
```suggestion
    use_cuda: bool = False,
```

`use_cuda` will always get a value ^

Reply:
Thanks, this has been adressed in 3ed1179.